models/intel/bert-small-uncased-whole-word-masking-squad-0002/accuracy-check.yml

```diff
@@ -0,0 +1,42 @@
+models:
+  - name: bert-small-uncased-whole-word-masking-squad-0002
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: bert_question_answering
+          start_token_logits_output: "output_s"
+          end_token_logits_output: "output_e"
+        inputs:
+          - name: "input_ids"
+            type: INPUT
+            value: "input_ids"
+          - name: "attention_mask"
+            type: INPUT
+            value: 'input_mask'
+          - name: 'token_type_ids'
+            type: INPUT
+            value: 'segment_ids'
+          - name: 'position_ids'
+            type: INPUT
+            value: 'position_ids'
+
+    datasets:
+      - name: squad_v1_1_msl384_mql64_ds128_lowercase
+        annotation_conversion:
+          vocab_file: vocab.txt
```

Comment:
looks confusing, please move this part to dataset_definitions.yml

Reply:
ok. 
Also I've made the same for previous model models/intel/bert-small-uncased-whole-word-masking-squad-0001